packages/devtools_app/lib/src/memory/memory_protocol.dart

```diff
@@ -18,7 +18,7 @@ import '../vm_service_wrapper.dart';
 class MemoryTracker {
   MemoryTracker(this.service);
 
-  static const Duration kUpdateDelay = Duration(milliseconds: 200);
+  static const Duration kUpdateDelay = Duration(milliseconds: 500);
```

Comment:
while you are here, refactor this constant to just be `updateDelay` to match current dart style.

Reply:
Done.